acceptance/helper.rb

```diff
@@ -264,6 +265,36 @@ def sleep_until_started(host)
     raise
   end
 
+  def is_bionic()
+    return test_config[:os_families].has_key? 'ubuntu1804-64-1'
+  end
+
+  def oldest_supported
+    # account for special case where bionic doesn't have builds before 5.2.4
+    return is_bionic ? '5.2.4' : '4.2.3.8'
+  end
+
+  def get_testing_branch(version)
+    branch_name = version[0..3] + 'x'
```

Comment:
This will break if we hit double digits in either the major or minor version(s). We could do something with regex like `/^(\d+.\d+.)(?:[\d]+.)*\d$/.match(version)[1] + 'x'`. As a side note, I use https://regex101.com/ when I'm playing around with constructing regular expressions.